### PR TITLE
[AIRFLOW-5485] - scheduler_job: replace == None with is

### PR DESCRIPTION
**Pull request by saurabhdhupar**
---
Issue link: [AIRFLOW-5485](https://issues.apache.org/jira/browse/AIRFLOW-5485)

Make sure to mark the boxes below before creating PR: [x]

- [x ] Description above provides context of the change
- [ x] Commit message/PR title starts with `[AIRFLOW-NNNN]`. AIRFLOW-NNNN = JIRA ID<sup>*</sup>
- [x ] Unit tests coverage for changes (not needed for documentation changes)
- [ x] Commits follow "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"
- [x ] Relevant documentation is updated including usage instructions.
- [x ] I will engage committers as explained in [Contribution Workflow Example](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#contribution-workflow-example).

<sup>*</sup> For document-only changes commit message can start with `[AIRFLOW-XXXX]`.

---
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).
Read the [Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines) for more information.

### CODE REVIEW

File: airflow/jobs/scheduler_job.py [hunk 1]
```diff
@@ -915,7 +915,7 @@ class SchedulerJob(BaseJob):
     }
     heartrate = conf.getint('scheduler', 'SCHEDULER_HEARTBEAT_SEC')
 
-    def __init__(
+    def  __init__(
```
Comment (Fokko) [inline airflow/jobs/scheduler_job.py:918]: I think this additional space will cause a lint error. Please revert this line.

**Comment by Fokko** [comment]
Thanks @saurabhdhupar for the contribution. Let's wait for the CI.

**Comment by mik-laj** [comment]
I did it earlier in this PR: https://github.com/apache/airflow/pull/7527

**Comment by ashb** [comment]
The other PR has been merged, so I'm closing this. Thanks for the PR though!

**Pull request closed without merging**